### DISCUSSION TITLE
Wash the mob...

### DISCUSSION THREAD

**Discussion by Fuchschr11** [humbertogontijo/homeassistant-roborock#346] [General]
How would I achieve to send a command to wash the mob, set the map to 3 and then goto a specific position to be carried upstairs.

Thank you.
Christian

**Comment by Fuchschr11** [comment]
I am a bit confused?? Is my question so obvious that everybody thinks it is so easy to do and nobody will answer. 
Or is it not possible?

Someone should have a clue. Please help.

Greetings
Christian

**Reply by poudenes** [reply]
I don't know. But for best result is wash the mob every day. If you don't do this the mob will smell horrible....Maybe that is reason nobody react on this... haha But yes. A reminder to do this is nice to have. I created a automation when Roborock is returning to dock send me a message to clean the mob
